pkg/hintrunner/zero/zerohint_ec.go

```diff
@@ -13,6 +13,71 @@ import (
 	"github.com/holiman/uint256"
 )
 
+// getHighLen calculates the highest bit length of `scalar_u.d2` and `scalar_v.d2`,
+// subtracts 1 from the result, and assigns it to `ids.len_hi`. 
+//
+// `newGetHighLenHint` takes three operanders as arguments:
+//   - `len_hi`: the variable that will store the result of the bit-length calculation.
+//   - `scalar_u.d2`: the first scalar value.
+//   - `scalar_v.d2`: the second scalar value.
+func newGetHighLenHint(len_hi, scalar_u, scalar_v hinter.ResOperander) hinter.Hinter {
+    return &GenericZeroHinter{
+        Name: "GetHighLen",
+        Op: func(vm *VM.VirtualMachine, ctx *hinter.HintRunnerContext) error {
+            //> ids.len_hi = max(ids.scalar_u.d2.bit_length(), ids.scalar_v.d2.bit_length())-1
+
+			scalarUAddr, err := scalar_u.GetAddress(vm)
+            if err != nil {
+                return fmt.Errorf("failed to resolve scalar_u address: %w", err)
+            }
+
+            scalarVAddr, err := scalar_v.GetAddress(vm)
+            if err != nil {
+                return fmt.Errorf("failed to resolve scalar_v address: %w", err)
+            }
+
+            scalarUValues, err := vm.Memory.ResolveAsBigInt3(scalarUAddr)
```

Comment:
do we really need to resolve `scalarU` and `scalarV` as BigInt3, given that we only need the third field of the struct?

Wouldn't it be more efficient to get the address, add an offset of 2 and retireve only one memory value?